pymoveit2/moveit2.py

```diff
@@ -1594,7 +1594,7 @@ def __init_move_action_goal(
         move_action_goal.request.allowed_planning_time = 0.5
         move_action_goal.request.max_velocity_scaling_factor = 0.0
         move_action_goal.request.max_acceleration_scaling_factor = 0.0
-        move_action_goal.request.cartesian_speed_end_effector_link = end_effector
+        move_action_goal.request.cartesian_speed_limited_link = end_effector
```

Comment:
If there are no other breaking changes, I would suggest the following logic in order to minimize the impact on users (NOT TESTED).

```suggestion
        # Note: Attribute was renamed in Iron (https://github.com/ros-planning/moveit_msgs/pull/130)
        if hasattr(move_action_goal.request, "cartesian_speed_limited_link"):
            move_action_goal.request.cartesian_speed_limited_link = end_effector
        else:
            move_action_goal.request.cartesian_speed_end_effector_link = end_effector
```